ops/helmfiles/dagster/forward_ports.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 # This script forwards port 8080 to the Kubernetes cluster running our Dagster install so you can access
 # the Dagit dashboard from your machine.
+set -e
```

Comment:
This is unrelated to the dataset changes, but:

Without this set, the gcloud k8s connection setup a few lines down can fail due to invalid creds (i.e., if you're attempting to connect to dev using your prod creds, or vice-versa). This led me to attempting to run prod workflows in dev. This change will cause the script to fail.